pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc

```diff
@@ -2,24 +2,24 @@
 = About {ProductShortName}
```

Comment:
@stoobie - this was the change suggested in FCCEUD-31 because it's what we had at the time, but both those attributes are the same today. As far as I can see, 'ProductShortName' is the one used in all our modules except the one about products, so at least today, my recommendation is for us to use the same one.